etna/lib/etna/clients/magma/formatting/models_csv.rb

```diff
@@ -255,7 +261,11 @@ def process_parent_config(models, row_processor)
             end
 
             row_processor.process(:parent_link_type, :model_name, :parent_model_name) do |parent_link_type, template|
-              reciprocal = models.find_reciprocal(model_name: template.name, link_attribute_name: template.parent)
+              reciprocal = models.find_reciprocal(
+                model_name: template.name,
+                link_attribute_name: template.name,
```

Comment:
Should this come from template.name? That means that we expect the attribute to be `parent <model_name>` - if we wanted the parent attribute to be called something else this would not work. This is another "soft" kind of rule about the data graphs that we should clarify.